.openshift-ci/post_tests.py

```diff
@@ -105,9 +106,11 @@ def __init__(
         collect_central_artifacts=True,
         check_stackrox_logs=False,
         artifact_destination_prefix=None,
+        add_test_result=True,
```

Comment:
I think keeping this in plural to match the `test_results` argument name would be better for consistency.